src/app/util/attribute-storage.cpp

```diff
@@ -883,7 +883,7 @@ uint16_t emberAfGetClusterServerEndpointIndex(EndpointId endpoint, ClusterId clu
         }
 
         // If this asserts, the provided fixedClusterServerEndpointCount doesn't match the app data model.
-        VerifyOrDie(adjustedEndpointIndex < fixedClusterServerEndpointCount);
+        VerifyOrDie(adjustedEndpointIndex <= fixedClusterServerEndpointCount);
```

Comment:
This doesn't seem right at first glance. The fix endpoints index are from 0 to (fix endpoints count -1)

If you have one fix endpoint you can conly have index 0. If you have 2 fixed endpoint your index are 0 or 1, etc

Reply:
Right, this was not the right fix.